andino_firmware/andino_firmware.ino

```diff
@@ -0,0 +1,4 @@
+/*
+ * Arduino IDE sketch file. This file is required to be able to compile and
+ * upload the Andino firmware using Arduino IDE.
+ */
```

Comment:
I tried to verifying it using the Arduino IDE but given that this file doesn't have any reference to setup or loop it fails. 
Are you missing here a include?

Reply:
Shouldn't we do something like the following: https://community.platformio.org/t/make-platformio-project-compatible-with-arduino-ide/3368/10

Reply:
That's weird, it works just fine on my end. When building, Arduino IDE is able to compile and link all .ino and .cpp files within the sketch folder, up to one level of folder nesting (see Arduino preprocessing process here: https://arduino.github.io/arduino-cli/0.33/sketch-build-process/#pre-processing).

I saw the link you provided, but in order to make it Arduino-agnostic, I'd rather not rename main.cpp to main.ino. I have followed this thorough tutorial instead: https://community.platformio.org/t/tutorial-for-creating-multi-cpp-file-arduino-project/5830/38.

I have tested the building process with both Arduino IDE 2.1.0 and 1.8.19. What version are you using?

The only required steps to build are:
- Open Arduino IDE.
- Go to File > Open.
- Select andino_firmware.ino.
- Click on verify.

Let me know how that goes, thanks!